backend-project/small_eod/institutions/models.py

```diff
@@ -1,43 +1,51 @@
 from django.core import validators
 from django.db import models
+
 from teryt_tree.models import JednostkaAdministracyjna
 
 from ..generic.models import TimestampUserLogModel
 from ..generic.validators import ExactLengthsValidator
 
 
 class AddressData(models.Model):
-    email = models.EmailField()
-    city = models.CharField(max_length=100)
-    epuap = models.CharField(max_length=100)
-    street = models.CharField(max_length=100)
-    house_no = models.CharField(max_length=100)
-    postal_code = models.CharField(max_length=100)
-    voivodeship = models.CharField(max_length=100)
+    email = models.EmailField(null=True, blank=True)
+    city = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
Na polach tekstowych nie należy stosować "null" (`null`), a zamiast tego przechowywać pusty tekst (`""`).

Reply:
CHAR and TEXT types are never saved as NULL by Django - przy takiej konfiguracji i tak bedzie przechowywane "". Jedyna roznica jest taka, ze teraz mozna recznie do bazy wrzucic null. Usune to